Base/res/apps/TextEditor.af

```diff
@@ -6,3 +6,6 @@ Category=Utilities
 [Icons]
 16x16=/res/icons/TextEditor16.png
 32x32=/res/icons/32x32/app-texteditor.png
+
+[Launcher]
+FileTypes=txt,md,html
```

Comment:
Please include `js`, `json` and `ini` here as well :)

Reply:
Wouldn't you also want cpp and h so itd open up in HackStudio? + HTM / the other html type file.

Reply:
I added these in, but for now it doesn't make any actual difference given the fallback handler defaults to TextEditor. Longer term, I think LaunchServer should become aware of the mapping between extension and MIME type, and then TextEditor can declare text/* so we don't have to keep adding extensions here.

HackStudio doesn't currently support launching with a given path through argv. When it does, it can declare which file types it wants to open.